tests/regtest.py

```diff
@@ -495,12 +503,16 @@ def setupFakeRoot(self, skip_proc):
 		# to own the root filesystem '/', thus let's chroot into /tmp,
 		# where we only bind mount the most important stuff from the
 		# root mount namespace
-		bind_dirs = ["/bin", "/sbin", "/usr", "/sys", "/dev", "/var"]
+		bind_dirs = ["/bin", "/sbin", "/sys", "/dev", "/var"]
 		# add any /lib32/64 symlinks whatever
 		bind_dirs.extend( glob.glob("/lib*") )
-		# /etc needs to be copied, we need to be able to write in
-		# there, to construct fake permissions config files
-		copy_dirs = ("/etc",)
+		# bind-mount everything in /usr except /usr/share/permissions
```

Comment:
Isn't that a bit overkill? I'm counting more than 350 bind mounts in my setup using this approach. I imagine it will be difficult finding your way through this setup when something goes awry.

How about mounting a tmpfs at `/usr/share/permissions` and keeping the copy approach? We can also bind mount r/w versions for single files like `mount --bind /tmp/permissions.local /etc/permissions.local`.

Reply:
We can only mount over existing files/directories. Today, `/usr/share/permissions` doesn't exist on any host systems, in the near future that'll be the case for the `/etc/permissions.*` files.